src/IntegrationTests/ITODBCInfo/test_odbc_info.cpp

```diff
@@ -67,7 +67,7 @@ int Ver1GEVer2(std::wstring ver_1_str, std::wstring ver_2_str) {
     if ((ver_1.size() == 0) || (ver_2.size() == 0))
         return -1;
 
-    size_t cnt = min(ver_1.size(), ver_2.size());
+    size_t cnt = std::min(ver_1.size(), ver_2.size());
```

Comment:
As I worked on GitHub Actions I found that:
- with min(..) it worked for Windows, but not Mac
- with std::min(..) it worked for Mac, but not for Windows.
Which is the annoying result of the std and min macro colliding in C / C++...
I used ((ver_1.size() < ver_2.size()) ? ver_1.size() : ver_2.size()); in place of this here to get rid of that issue.

Reply:
applied suggestion